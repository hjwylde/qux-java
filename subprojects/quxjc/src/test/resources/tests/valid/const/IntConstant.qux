package valid.const

int FOO is 5
int BAR is 5
int BAZ is -5

int f(int a):
    return a + FOO

int g(int a):
    return a + BAZ

void main([str] args):
    print FOO
    print BAR
    print BAZ
    print FOO == BAR
    print FOO == BAZ
    print BAR == BAZ
    print f(FOO)
    print g(FOO)

