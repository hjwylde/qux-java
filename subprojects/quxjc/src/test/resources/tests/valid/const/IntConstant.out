5
5
-5
true
false
false
10
0
